helmfile.d/helmfile-09.init.yaml

```diff
@@ -24,12 +24,12 @@ releases:
       pkg: knative
     values:
       - ../values/knative/knative-serving-raw.gotmpl
-  - name: gatekeeper-operator-config
+  - name: gatekeeper-operator-artifacts
     installed: {{ $c | get "gatekeeper-operator.enabled" false }}
     namespace: gatekeeper-system
     labels:
       pkg: gatekeeper-operator
-    <<: *default
+    <<: *raw
```

Comment:
this won't be straightforward as usual. You will have to generate in a helmfile pre-hook the final constraint yamls (maybe somewhere in `/tmp`): `values/opa/*.gotmpl` (see below comments for explanation)

then you have all the statics that need to be bundled in `values/opa/gatekeeper-raw.gotmpl` (rename `values/gatekeeper-operator`to `values/opa`and rename `gatekeeper-operator.gotmpl` to `gatekeeper.gotmpl`)